docs/build/inferencing.md

```diff
@@ -31,7 +31,7 @@ Basic CPU build
 
 * Install [Python 3.x](http://python.org/).
 
-* Install [cmake-3.24](https://cmake.org/download/) or higher.
+* Install [cmake-3.26](https://cmake.org/download/) or higher.
```

Comment:
I assume we've good reasons to keep upgrading cmake.

Reply:
If we are not willing to support a lot of different cmake versions, the best version to support is the latest one. For example, if we must choose one from "VS 2019" and "VS 2022" to support, the later one seems better. 
And we do use some new features that are only available in recent cmake versions.